packages/app-content-pages/README.md

```diff
@@ -19,16 +19,16 @@ cp env-default .env
 
 To get the credentials, go to https://app.contentful.com/spaces/jt90kyhvp0qv/api/keys (login credentials are in Passbolt), and copy the Space ID and Content Delivery API token from "Testing".
 
-### Running in development
+## Running in development
 
-#### Docker
+### Docker
 
-- `docker-compose up` to run a server on http://localhost:3000.
+- `docker-compose up` to run a server on http://localhost:3000. The `--build` flag can be used to build the container. This builds and runs a local image which matches the Jenkins build except for running behind a proxy.
 - `docker-compose down` to stop the dev server.
 - `docker-compose run --rm content-pages test` to run the tests.
 - `docker-compose --rm content-pages start` to run a production build on http://localhost:3000.
```

Comment:
What does production mean here in this context? Panoptes production?

Reply:
Webpack production build using `next build`. I think it’s set to use the production API too, but that can be changed if needed.

Reply:
This came up during the conversation about specifying the build environment for #1887. The classifier ignores `NODE_ENV` and `PANOPTES_ENV` from the shell and uses its own values, hardcoded into the webpack configs.
https://github.com/zooniverse/front-end-monorepo/blob/922b76f637ea45ad398c66c11dc555b07a1c2f39/packages/lib-classifier/webpack.dev.js#L5-L9
https://github.com/zooniverse/front-end-monorepo/blob/922b76f637ea45ad398c66c11dc555b07a1c2f39/packages/lib-classifier/webpack.dist.js#L4-L8

The NextJS apps  use the values set in the shell, so a local production build against the staging API (for testing changes in production maybe?)
```sh
NODE_ENV=production PANOPTES_ENV=staging yarn bootstrap
```
would build Next apps that use Panoptes staging, but the classifier build would still use Panoptes production.

This would only be a problem when specifying an environment for local testing, eg. #1915. Jenkins sets `NODE_ENV` and `PANOPTES_ENV` to production so won't cause any conflicts between the NextJS apps and the classifier.